tests/func/test_pgconf.py

```diff
@@ -475,3 +476,52 @@ def test_16_get_pgconf_configuration_category_ko_406(self):
             }
         )
         assert status == 406
+
+    @pytest.mark.skipif("ENV['pg']['version'] < 12")
```

Comment:
This should not be a string I think. As is, the condition is always true (and the test always skipped).

Reply:
Curiously, it works well.
Without the parenthethis I get an error (KeyError: 'pg') when pytest is reading the file to search for tests.
It looks like with parenthethis, the string is evaluated only at the time the test is run. And the test is effectively skipped for version 11 and under.